.cg-deploy/manifests/dev/manifest-api-dev.yml

```diff
@@ -0,0 +1,20 @@
+---
+applications:
+- name: open-forest-platform-api-dev
+  routes:
+    - route: fs-intake-api-staging.app.cloud.gov
```

Comment:
this route says `staging` because that is what our EAuth integration requires, but it is a different configuration than the one we use in our staging environment.